docs/usage/examples/self-hosting.md

```diff
@@ -396,3 +396,35 @@ The logging level output is controlled by the Bunyan logging library.
 |    40 | warn    |
 |    50 | error   |
 |    60 | fatal   |
+
+## Self-signed TLS/SSL certificates
+
+Renovate and invoked helper programs (e.g. Git, npm) use a secure TLS connection (e.g. HTTPS) to connect to source code platforms and dependency hosts.
+If the systems use any self-signed certificates or certificate authorities then Renovate needs to be configured to trust these additional certificates.
```

Comment:
```suggestion
Renovate and invoked helper programs (e.g. Git, npm) use a secure TLS connection (e.g. HTTPS) to connect to remote source code and dependency hosts.
If the remote hosts use any self-signed certificates or certificate authorities then Renovate needs to be configured to trust these additional certificates.
```
@HonkingGoose maybe even rephrasing it in both lines to call them "hosts" makes it clearer about what we're talking.

Reply:
I like it, it makes the connection between the sentences clearer, now that they both have the word `remote` in them.